.github/workflows/flutter.yaml

```diff
@@ -0,0 +1,38 @@
+name: Dart
+
+on:
+  schedule:
+    # “At 00:00 (UTC) on Sunday.”
+    - cron: '0 0 * * 0'
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: clone the repo
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: install Flutter sdk
+        uses: subosito/flutter-action@dbf1fa04f4d2e52c33185153d06cdb5443aa189d
+        with:
+          channel: 'beta'
+
+      - name: content
```

Comment:
You might want something more descriptive for the step title (this is printed out when the action runs).

I don't know that you need `pwd` or `ls` below (I imagine this was here for debugging the workflow?); they are however not hurting anything.

Reply:
Cleaned up.